src/distro/distro.py

```diff
@@ -1245,39 +1260,31 @@ def _distro_release_info(self) -> Dict[str, str]:
                 # sure about the *-release files. Check common entries of
                 # /etc for information. If they turn out to not be there the
                 # error is handled in `_parse_distro_release_file()`.
-                basenames = [
-                    "SuSE-release",
-                    "arch-release",
-                    "base-release",
-                    "centos-release",
-                    "fedora-release",
-                    "gentoo-release",
-                    "mageia-release",
-                    "mandrake-release",
-                    "mandriva-release",
-                    "mandrivalinux-release",
-                    "manjaro-release",
-                    "oracle-release",
-                    "redhat-release",
-                    "rocky-release",
-                    "sl-release",
-                    "slackware-version",
-                ]
+                basenames = _DISTRO_RELEASE_BASENAMES
             for basename in basenames:
                 if basename in _DISTRO_RELEASE_IGNORE_BASENAMES:
                     continue
                 match = _DISTRO_RELEASE_BASENAME_PATTERN.match(basename)
-                if match:
-                    filepath = os.path.join(self.etc_dir, basename)
-                    distro_info = self._parse_distro_release_file(filepath)
-                    if "name" in distro_info:
-                        # The name is always present if the pattern matches
-                        self.distro_release_file = filepath
-                        distro_info["id"] = match.group(1)
-                        if "cloudlinux" in distro_info["name"].lower():
-                            distro_info["id"] = "cloudlinux"
-                        return distro_info
-            return {}
+                if not match:
+                    continue
+                filepath = os.path.join(self.etc_dir, basename)
+                distro_info = self._parse_distro_release_file(filepath)
+                # The name is always present if the pattern matches.
+                if "name" not in distro_info:
+                    continue
+                self.distro_release_file = filepath
+                break
+            else:
+                return {}
+
+        if match is not None:
+            distro_info["id"] = match.group(1)
+
+        # manually set id for CloudLinux
+        if "cloudlinux" in distro_info.get("name", "").lower():
+            distro_info["id"] = "cloudlinux"
```

Comment:
@HorlogeSkynet let's add a comment here that…
- this is for CloudLinux <7, not CloudLinux in general and that
- we're (not overwriting things here but) adding something missing.

My undererstanding is that `"name"` will be the only field set (to the single free-form line from file `/etc/redhat-release`) for CloudLinux 5 and 6 and this adds an ``"id"` field to be "a little less lacking". My references are:

- https://github.com/python-distro/distro/blob/master/tests/resources/distros/cloudlinux6/etc/redhat-release
- https://github.com/python-distro/distro/blob/master/tests/resources/distros/cloudlinux7/etc/os-release
- https://github.com/python-distro/distro/blob/697b2dbb20ab12cb86becf57f062645de723d5ac/src/distro/distro.py#L1326

What do you think?